tests/vision/checks/model_evaluation/single_dataset_performance_test.py

```diff
@@ -95,8 +95,8 @@ def test_segmentation_many_scorers(segmentation_coco_train_visiondata, trained_s
     result = check.run(segmentation_coco_train_visiondata, trained_segmentation_deeplabv3_mobilenet_model,
                        device=device, with_display=False).value
 
-    assert_that(result[result['Metric'] == 'dice']['Value'].mean(), close_to(0.655, 0.001))
-    assert_that(result[result['Metric'] == 'dice_macro']['Value'], close_to(0.655, 0.001))
+    assert_that(result[result['Metric'] == 'dice']['Value'].mean(), close_to(0.649, 0.001))
+    assert_that(result[result['Metric'] == 'dice_macro']['Value'], close_to(0.649, 0.001))
```

Comment:
Why did the result change?

Reply:
Change in coco-segmentation by ultralytics